docs/tasks/istio-installation.md

```diff
@@ -1,37 +1,105 @@
 ---
-title: Installing Istio
-headline: Installing Istio
+title: Istio Installation
+headline: Istio Installation
 sidenav: doc-side-tasks-nav.html
 bodyclass: docs
 layout: docs
 type: markdown
 ---
 {% capture overview %}
-This task shows how to install Istio in a Kubernetes cluster. You'll learn
+This page shows how to get started using Istio in a Kubernetes cluster. You'll learn
 how to install Istio and get it initially configured and running.
 {% endcapture %}
 
 {% capture prerequisites %}
-* Do this.
-* Do this too.
+* The following instructions assume you have access to a Kubernetes cluster. To install Kubernetes locally, try [minikube](https://github.com/kubernetes/minikube).
+* If you are using GKE, please make sure you are using static client certificates before fetching cluster credentials:
+```bash
+gcloud config set container/use_client_certificate True
+gcloud container clusters get-credentials cluster-2 --zone us-central1-a --project istio-demo
+```
+* Ensure curl is present, or clone the github [istio](https://github.com/istio/istio) repository to access the files locally.
 {% endcapture %}
 
 {% capture steps %}
-## Doing ...
+## Install Istio on an existing Kubernetes cluster
+For alpha release, Istio must be installed in the same Kubernetes namespace as the applications. Instructions below will deploy Istio in the "default" namespace. They can be modified for deployment in a separate namespace.
 
-1. Do this.
-1. Do this next. Possibly read this [related explanation](...).
+1. Source Istio configuration file:
```

Comment:
Frank's earlier documentation had this for the Istio.version file..
The [istio.VERSION](https://github.com/istio/istio/blob/master/istio.VERSION) file includes the download location of three OS-specific binaries: `istioctl-osx`, `istioctl-win.exe`, `istioctl-linux` targeted at Mac, Windows and Linux users respectively. Download the tool appropriate to your platform. For example, when running istioctl on a Mac, run the following commands:
 
 ```bash
 source ./istio.VERSION # set ISTIOCTL env variable
 curl ${ISTIOCTL_URL}/istioctl-osx > /usr/local/bin/istioctl
chmod +x /usr/local/bin/istioctl
 ```
  > Note: If you already have a previously installed version of `istioctl`, make sure that
  > it is compatible with the manager image used in `demos/kubernetes/istio-manager.yaml`.
  > If in doubt, download again or add the `--tag` option when running `istioctl kube-inject`.
  > Invoke `istioctl kube-inject --help` for more details.

These seem pretty important, esp the bit about compatibility stuff.